boards/adi/apard32690/apard32690_max32690_m4.dts

```diff
@@ -53,6 +53,35 @@
 		led2 = &green_led;
 		sw0 = &usr_btn;
 	};
+
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio2 0 0>,	/* A0 */
+			   <1 0 &gpio2 1 0>,	/* A1 */
+			   <2 0 &gpio2 2 0>,	/* A2 */
+			   <3 0 &gpio2 3 0>,	/* A3 */
+			   <4 0 &gpio2 4 0>,	/* A4 */
+			   <5 0 &gpio2 5 0>,	/* A5 */
+			   <6 0 &gpio2 16 0>,	/* D0 */
+			   <7 0 &gpio2 14 0>,	/* D1 */
+			   <8 0 &gpio2 15 0>,	/* D2 */
+			   <9 0 &gpio2 13 0>,	/* D3 */
+			   <10 0 &gpio0 8 0>,	/* D4 */
+			   <11 0 &gpio0 7 0>,	/* D5 */
+			   <12 0 &gpio1 25 0>,	/* D6 */
+			   <13 0 &gpio1 24 0>,	/* D7 */
+			   <14 0 &gpio1 31 0>,	/* D8 */
+			   <15 0 &gpio1 30 0>,	/* D9 */
```

Comment:
Seems P1.24 and P1.25  need to be switched. Could you double check pin mapping: https://wiki.analog.com/_media/resources/eval/user-guides/02-073637-01-c.pdf

![image](https://github.com/user-attachments/assets/1c94dffa-0635-413e-9751-f298c6b78eda)

Reply:
Good catch. I had rev A of the schematic and you have rev C. I reviewed all of the pins again and found a couple more changes.